homework/style.css

```diff
@@ -1,3 +1,135 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  font-size: 1.5rem;
+  padding: 5%;
+  background-color: pink;
+  color: black;
+}
+
+body {
+  width: 768px;
```

Comment:
It's better to use a max-width in pixels and a width in percentage on the body to make the site even more responsive.

Reply:
Thank you for this advice, i used as you suggest.